kernel/build/config-arm64

```diff
@@ -1,6 +1,6 @@
 #
```

Comment:
this is `make olddefconfig` right? just to be safe

Reply:
`make kernel-olddefconfig`